### ISSUE TITLE
BLU enfeebling spells (Sheep Song, Temporal Shift, Actinic Burst, ect.) should not have a dStat modifier

### ISSUE THREAD

**Issue by Anonyy89** [EdenServer/community#5214]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted.
- [x] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [x] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Version 30181205_0

**tl;dr - BLU enfeebling spells should not be using dstat for magic accuracy. If they are on Eden then information on which spells and which dstat need to be provided so that retail testing can be done to either debunk or solidify the information.**

BLU enfeebling spells (Sheep Song, Temporal Shift, Actinic Burst, ect.) should not have a dStat modifier. Meaning that if I cast "Sheep Song" none of my stats should be adding magic accuracy to the spell. Not MND, INT, or CHR. I believe it is suppose to use pure magic accuracy only.

While I do not have concrete evidence suggesting so, **there is an overwhelming amount of "hearsay" that suggest BLU enfeebling spells do not use a dstat modifier.** Again that isn't the most credible evidence but I think once its coupled with the fact **that there is almost NO evidence or even "hearsay" that suggests that these spells should use dstat,** it becomes a much more solid and arguementive case.

One main source is the entire page 401 of the following thread: https://www.ffxiah.com/forum/topic/30626/the-beast-within-a-guide-to-blue-mage/401/#3561928. Seeing as this thread (which is dedicated purely to BLU) was started way back in 2012 by "Prothescar" (who is a very reliable source on BLU things) and has been maintained and updated as recently as a few hours ago. I feel like it is a solid source of information. On the mentioned page is more confirmation from different sources that BLU enfeebling spells do not use a dstat.

Another source is listed on the "Calculating BLU magic damage page" (https://ffxiclopedia.fandom.com/wiki/Calculating_Blue_Magic_Damage). This page breaks down, in great detail, information about physical/magical BLU spells that deal damage. The most relevant information here is that it breaks down what Stat(s) each spell uses as well as the formula. At the very bottom of the page "Enfeebling BLU spells" are listed, with no formulas or stats or anything. This leads me to believe that those particular spells (and ones similar to them that arent on the list) use the standard Magic Accuracy formula (https://ffxiclopedia.fandom.com/wiki/Magic_Accuracy). Only issue is there are no stats/dstats listed ANYWHERE for those enfeebling spells. In all the time this job has been active in retail, you would think that if these spells used dstat it would have been figured out by now or we would at least have some sort of assumption for their dstats, but instead we have nothing.

This is something that could be easily tested in retail, but because doing so would be incredibly time consuming, I think any testing should be put off until it is determined 100% that enfeebling BLU spells on Eden use dstat, and which stat is used for which spell. **Once we have that information retail testing could be started.**

I'll include any other sources of evidence I have in the comments as I gather and organize them.

**Comment by Anonyy89** [comment]
I understand that if everything I posted above is true, that this would mean BLU enfeebling spells are assumed to have a 0 dstat modifier in the Magic Accuracy formula. And if that were the case then those spells would have an extremely hard time landing.

I think the "Magic Hit Rate" page hints at the solution to this (https://ffxiclopedia.fandom.com/wiki/Magic_Hit_Rate).
**_Keep in mind that all spells don't have the same magic accuracy, or equivalently, monsters/players have different magic evasions to different spells, even without using barspells. For instance, higher tiers of elemental magic resist less than lower tiers (this is unquantified)._**

Maybe BLU enfeebling spells have a flat bonus to "not resisting as often". I honestly don't a/the solution to BLU enfeebling spells not having a dstat, I just know that they should not have that modifier.

**Comment by 59blargedy** [comment]
there are some concrete tests that show dstat = macc, the comments after the post linking the tests seem to firmly show that the effect was proven but the link referenced is sadly broken and wayback machine didnt work
https://www.ffxiah.com/forum/topic/9144/stun-locking/3/#512937
also two instances of jp testing are linked twice on the first page of the same thread that suggests a relation (but also makes some odd claims)
three tests that claim a relation, but 0 tests proving a lack of effect is where we sit at currently. there's numerous mentions of dstat being a part of macc, but i took those as assumptions since players in general associate dstat to macc. i mentally throw those and any lack of mention out the window.

**Comment by Anonyy89** [comment]
> there are some concrete tests that show dstat = macc, the comments after the post linking the tests seem to firmly show that the effect was proven but the link referenced is sadly broken and wayback machine didnt work https://www.ffxiah.com/forum/topic/9144/stun-locking/3/#512937 also two instances of jp testing are linked twice on the first page of the same thread that suggests a relation (but also makes some odd claims) three tests that claim a relation, but 0 tests proving a lack of effect is where we sit at currently. there's numerous mentions of dstat being a part of macc, but i took those as assumptions since players in general associate dstat to macc. i mentally throw those and any lack of mention out the window.

It looks like that entire thread and even the jp testing is all for headbutt. This issue is for BLU enfeebling spells not physical damage spells.

**Comment by 59blargedy** [comment]
the tests are for the enfeebling effect landing, not the physical hit, and are discussing dstat and macc.

**Comment by Anonyy89** [comment]
> the tests are for the enfeebling effect landing, not the physical hit, and are discussing dstat and macc.

I think you might be missing the point of this whole post. This is for BLU enfeebling spells only. Not physical/magical spells that deal damage. Not for additional effects. and especially not for headbutt which is a rare exception in and of itself.

**Comment by Anonyy89** [comment]
> I understand that if everything I posted above is true, that this would mean BLU enfeebling spells are assumed to have a 0 dstat modifier in the Magic Accuracy formula. And if that were the case then those spells would have an extremely hard time landing.
> 
> I think the "Magic Hit Rate" page hints at the solution to this (https://ffxiclopedia.fandom.com/wiki/Magic_Hit_Rate). **_Keep in mind that all spells don't have the same magic accuracy, or equivalently, monsters/players have different magic evasions to different spells, even without using barspells. For instance, higher tiers of elemental magic resist less than lower tiers (this is unquantified)._**
> 
> Maybe BLU enfeebling spells have a flat bonus to "not resisting as often". I honestly don't a/the solution to BLU enfeebling spells not having a dstat, I just know that they should not have that modifier.

To elaborate on this more, assuming BLU enfeebling spells work in the same manner as Ninjutsu, what does Ninjutsu have/use in place for his dstat modifier that keeps it from being constantly resisted despite not having a dstat? Maybe Ninjutsu and BLU enfeebling spells both have a bonus to "Magic Hit Rate". 

I really think starting from square 1 and checking if BLU spells are even using a dstat modifier on Eden, and then figuring out which stat is which spell if so, is the best course of action to start with. It would let me and anyone else who may be wondering about this be able to perform tests on retail and answer alot of questions. I think it'll have a big domino effect.

**Comment by Steam-arch** [comment]
Im pretty sure on eden blu enfeebling spells such as Filamented Hold etc always have MND as a dstat modifier. (Someone correct me if im wrong, atleast esunii told me to stack MND so ill shift blame to him )

Maybe easiest to just go on retail, pick 1-3 blu enfeebling spells and check if their macc is affected by MND/CHR/other

**Comment by falzum** [comment]
A variable to consider in your cape terrigan testing is level correction. You're 10 levels above the mob. That's an incredible boost to magic accuracy right out the gate.

Using melee for example. Every level below the mob you get -4acc, but every level above you get +4.

Thats +40 accuracy just from your level

I dont know exactly the magic accuracy level correction formula but it's definitely influencing your results

**Comment by Anonyy89** [comment]
> A variable to consider in your cape terrigan testing is level correction. You're 10 levels above the mob. That's an incredible boost to magic accuracy right out the gate.
> 
> Using melee for example. Every level below the mob you get -4acc, but every level above you get +4.
> 
> Thats +40 accuracy just from your level
> 
> I dont know exactly the magic accuracy level correction formula but it's definitely influencing your results

I'd have to double check but Im almost certain level correction always works as a penalty to the lower lvl unit in the equation, and never as a bonus to the higher lvl unit. Meaning like just because something is higher lvl than me doesn't mean it gets a bonus to its accuracy to hit me. I just have a penalty to my accuracy to hit it (since Im lower lvl). The higher lvl mob doesn't get an evasion bonus, I get a penalty to accuracy.

Those mobs in Cape Terrigan have a penalty to hit me, I don't have a bonus to hit them.

**Comment by Anonyy89** [comment]
> Im pretty sure on eden blu enfeebling spells such as Filamented Hold etc always have MND as a dstat modifier. Maybe im wrong though, propably, 10% sure on the statement now that I question myself.
> 
> Maybe easiest to just go on retail, pick 1-3 blu enfeebling spells and check if their macc is affected by MND/CHR/other

Yeah this is my plan. But before I can do this I have to be certain BLU enfeebling spells on Eden have a dstat modifier and I have to know what stat they use. Once I know that, I can make comparisons in retail and run the same test I did here on Eden and see how the results match up.

Im hoping the quick test I've done is enough to shed some light on this.
